cms/settings/functional_test.py

```diff
@@ -1,10 +1,3 @@
-import copy
-
 from .test import *  # noqa: F403  # pylint: disable=wildcard-import,unused-wildcard-import
 
-DATABASES = {
-    "default": dj_database_url.config(default="postgres://ons:ons@localhost:15432/ons"),  # noqa: F405
-}
```

Comment:
NB: the port here is 15432 as opposed to 5432 used elsewhere, you might want to keep this or otherwise take this into account as this is why your functional tests are currently failing.
https://github.com/ONSdigital/dis-wagtail/blob/30380c1a236b3df0590f6acf3d3d959c08b3b94d/functional_tests/docker-compose.yml#L6-L7

Reply:
Oh wow, nice catch!